cloudaeye/manifest.json

```diff
@@ -0,0 +1,76 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "4a0bb068-63fa-4cc2-a557-0452e8dca689",
+  "app_id": "cloudaeye",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "uninstallation": "README.md#Uninstallation",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Troubleshoot and resolve problems in seconds, remove toil, and earn customer trust.",
+    "title": "CloudAEye",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "CloudAEye Kosal Dashboard",
+        "image_url": "images/cloudAEyeKosalDashboard.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Automated Root Cause Analysis",
+        "image_url": "images/root-cause-analysis.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Notifications with Rich Context",
+        "image_url": "images/notifications.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Centralized app analysis",
+        "image_url": "images/global-context.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Assistant",
+        "image_url": "images/assistant.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::AI/ML",
+      "Category::Kubernetes",
+      "Category::Incidents",
+      "Category::Automation",
+      "Category::Issue Tracking",
+      "Offering::Integration",
+      "Queried Data Type::Metrics",
+      "Queried Data Type::Logs",
+      "Queried Data Type::Traces",
+      "Queried Data Type::Events",
+      "Submitted Data Type::Events",
+      "Supported OS::Any"
+    ]
+  },
+  "author": {
+    "support_email": "support@cloudaeye.com",
+    "sales_email": "support@cloudaeye.com",
+    "homepage": "https://www.cloudaeye.com",
+    "name": "CloudAEye"
+  },
+  "assets": {
+    "integration": {
+      "auto_install": true,
+      "source_type_id":"26690330",
+      "source_type_name": "CloudAEye",
+      "configuration": {},
+      "events": {
+        "creates_events": true
+      },
+    "dashboards": {
+      "CloudAEye Kosal Dashboard": "assets/dashboards/CloudAEye_overview.json"
+    }
```

Comment:
Dashboard needs to be moved outside the `integration` but stay inside `assets`. Thanks